src/stuf/stuf_zds/templates/stuf_zds/soap/creeerZaak.xml

```diff
@@ -1,5 +1,5 @@
-{% load l10n %}
-<ZKN:zakLk01
+{% extends "stuf/soap_envelope.xml" %}{% load stuf l10n %}
+{% block body %}<ZKN:zakLk01
         xmlns:StUF="http://www.egem.nl/StUF/StUF0301"
```

Comment:
namespace mess - StUF-BG defines them on the root element, while StUF-ZDS defines them on the children? Why this inconsistency?

Reply:
use them on domain root element

Reply:
Resolved via 0126688d73541bbe5ff157b1114d35cc71426dec